app/controllers/administrative-units/administrative-unit/core-data/edit.js

```diff
@@ -195,9 +195,13 @@ export default class AdministrativeUnitsAdministrativeUnitCoreDataEditController
       yield structuredIdentifierKBO.save();
       yield identifierKBO.save();
 
-      structuredIdentifierSharepoint = setEmptyStringsToNull(
-        structuredIdentifierSharepoint
-      );
+      // FIXME: If uncommented existing SharePoint identifier is not removed
```

Comment:
**Question:** Any error message in the console when you try to save with null value?

Reply:
Nope. As mentioned in the standup, I looked at the logs from the resources service and the behaviour is the same as when removing other fields, such as a phone number. The difference is that with identifiers you have the extra layer of `structuredIdentifier` and `localId`. The resources service does not delete the triple for the `localId` but only deletes and re-inserts the triple for `structuredIdentifier`.